CHANGELOG.md

```diff
@@ -1,12 +1,80 @@
 # Changelog
 
-## Unreleased
+## v6.0.0-rc.2 - 2017-11-14
 
-- Update the contribution guideline to note that updating the changelog is required. (#5586)
+### Breaking changes
+
+- Skip null-element check for collections with a value-type element type (#5432)
+- Make 'AllSigned' execution policy require modules under $PSHome to be signed (#5511)
+
+### Engine updates and fixes
+
+- Update powershell to use `2.0.4` dotnet core runtime. (#5677)
+- Remove references to the old executable `powershell` or `powershell.exe` (#5408)
+
+### General cmdlet updates and fixes
+
+- Remove unncessary check for Paths.count > 0, in the `*-FileCatalog` CmdLets, as there is code later to use the current (#5596)
+- Use explicit libpsl-native binary name for dllimport. (#5580)
+
+### Build and Packaging Improvements
+
+- Fix Get-EnvironmentInformation to properly check for CoreCLR (#5592) (Thanks @markekraus!)
+- Make Travis CI use `libcurl+openssl+gssapi` (#5629) (Thanks @markekraus!)
+- Disambiguate icon for daily builds on Windows (#5467) (Thanks @bergmeister!)
+- Fix Import-CliXml tests which still use `powershell` instead of `pwsh` and make sure it fails if it regresses (#5521) (Thanks @markekraus!)
+- Update port used to WebCmdLet test which broke due to a change in AppVeyor (#5520) (Thanks @markekraus!)
+- Clean up use of 'Runspaceconfiguration' from comments and xunit test code (#5569) (Thanks @Bhaal22)
 - Remove Pester as a module include with the PowerShell Packages.
   In the future, you should be able to add it by running `Install-Module Pester`. (#5623, #5631)
-- Make Travis CI use `libcurl+openssl+gssapi` (#5629, @markekraus)
 - Refactor `New-UnixPackaging` into functions to make the large function more readable. (#5625)
+- Make the experience better when start-pspester doesn't find pester (#5673)
+- Update packaging and release build scripts to produce zip packages for win-arm/win-arm64 (#5664)
+- Enable `Install-Debian` to work with VSTS Hosted Linux Preview (#5659)
+- Add linux-arm tarball package to release build (#5652, #5660)
+- Enable building for win-arm and win-arm64 (#5524)
+- Make macOS package require 10.12 or newer (#5649, #5654)
+- Update signing subjects to something meaningful (#5650)
+- Make `New-UnixPackage` more readable (#5625)
+- Update `PowerShellGet` tests to validate the new install location of AllUsers scope. (#5633)
+- Increase reliability of flaky test that fails intermittently in CI (#5641)
+- Exclude markdown files from 'Pester' folder from the Markdown meta test (#5636)
+- Run tests for Windows installer only on Windows (#5619)
+- Supress the expected errors from Select-Xml tests (#5591)
+- Make the -SslProtocol tests pending (#5605)
+- Add retry logic to prerequisite URL and output URL on failure so you can more easily troubleshoot (#5601, #5570)
+- Replace HttpListener Response Tests with WebListener (#5540)
```

Comment:
Was this done by `markekraus`? If so, we should mention him.

Reply:
resolved